Source/SwiftLintFramework/Rules/Idiomatic/ToggleBoolRule.swift

```diff
@@ -0,0 +1,35 @@
+import SourceKittenFramework
+
+public struct ToggleBoolRule: ConfigurationProviderRule, OptInRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static var description = RuleDescription(
+        identifier: "toggle_bool",
+        name: "Toggle Bool",
+        description: "Prefer `Bool.toggle()` over `someBool = !someBool`.",
```

Comment:
should we use `someBool.toggle()` instead of `Bool.toggle()` to be more explicit?

Reply:
Yah, I think that makes more sense. I've also updated the Changelog to match.